packages/falso/src/lib/rgb.ts

```diff
@@ -2,6 +2,19 @@ function randomValue() {
   return Math.floor(Math.random() * (255 + 1));
 }
 
-export function rgb() {
-  return `rgb(${randomValue()}, ${randomValue()}, ${randomValue()})`;
+function randomFloat(min: number, max: number) {
```

Comment:
Can you please replace the implementation of our `randomFloat` with this and use an object param? e.g `randomFloat({ min, max })`